src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -68,25 +72,27 @@ public float GetPressureScore(SelectionPressure pressure, Patch patch, Species s
         return cached;
     }
 
-    public EnergyBalanceInfo GetEnergyBalanceForSpecies(MicrobeSpecies species, IBiomeConditions biomeConditions)
+    public EnergyBalanceInfoSimple GetEnergyBalanceForSpecies(MicrobeSpecies species, IBiomeConditions biomeConditions)
     {
         // TODO: this gets called an absolute ton with the new auto-evo so a more efficient caching method (to allow
         // different species but with same organelles to be able to use the same cache value) would be nice here
         var key = (species, biomeConditions);
 
-        if (cachedEnergyBalances.TryGetValue(key, out var cached))
+        if (cachedSimpleEnergyBalances.TryGetValue(key, out var cached))
         {
             return cached;
         }
 
         var maximumMovementDirection = MicrobeInternalCalculations.MaximumSpeedDirection(species.Organelles);
 
+        cached = new EnergyBalanceInfoSimple();
```

Comment:
Could you add a TODO comment here like `// TODO: check if caching instances of these objects would be better than always recreating`? I expect there to be no big difference but for future reference I'd like a comment here pointing out this possibility.